catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/AsyncFileAlterationObserver.java

```diff
@@ -120,10 +115,20 @@ private AsyncFileAlterationObserver(AsyncFileEntry entry, ObjectPersistentStore
   public void initialize() throws IllegalStateException {
     initChildEntries(rootFile);
     serializer.store(rootFile.getName(), rootFile);
+
+    if (LOGGER.isDebugEnabled()) {
```

Comment:
❓ Did you realize that you will only startup the timer thread if, at init time, debug is enabled?

Reply:
Good point. The timertask checks for debug before working anyway, I'll just remove this one